libqtile/confreader.py

```diff
@@ -53,15 +54,21 @@ class Config:
         "wmname",
     ]
 
-    def __init__(self, file_path=None, **settings):
+    def __init__(self, file_path=None, kore=None, **settings):
         """Create a Config() object from settings
 
         Only attributes found in Config.settings_keys will be added to object.
         config attribute precedence is 1.) **settings 2.) self 3.) default_config
         """
+        self.file_path = file_path
+        self.kore = kore
+        self.update(**settings)
+
+    def update(self, fake_screens=None, **settings):
```

Comment:
It would be nice to guard `fake_screens` as a keyword-only argument.

Reply:
Sounds good!